Code/Framework/AzCore/AzCore/Component/ComponentApplicationLifecycle.cpp

```diff
@@ -77,11 +77,12 @@ namespace AZ::ComponentApplicationLifecycle
         }
         auto eventNameRegistrationKey = FixedValueString::format("%.*s/%.*s", AZ_STRING_ARG(ApplicationLifecycleEventRegistrationKey),
             AZ_STRING_ARG(eventName));
-        auto lifecycleCallback = [callback = AZStd::move(callback), eventNameRegistrationKey](AZStd::string_view path, Type type)
+        auto lifecycleCallback = [callback = AZStd::move(callback), eventNameRegistrationKey](
```

Comment:
There is now an Android compile warning now due to the Type alias on line 62 in this file being unused
https://jenkins.build.o3de.org/job/O3DE/job/PR-11028/.

You will need to remove that line to fix the compile warning.